src/sentry/models/groupassignee.py

```diff
@@ -160,18 +170,16 @@ def assign(self, group, assigned_to, acting_user=None):
             )
 
         if affected:
-            activity = Activity.objects.create(
-                project=group.project,
-                group=group,
-                type=Activity.ASSIGNED,
+            Activity.objects.create_group_activity(
+                group,
+                ActivityType.ASSIGNED,
                 user=acting_user,
                 data={
                     "assignee": str(assigned_to.id),
                     "assigneeEmail": getattr(assigned_to, "email", None),
                     "assigneeType": assignee_type,
                 },
             )
-            activity.send_notification()
```

Comment:
Things like this make this PR really difficult to follow. Did this move somewhere else? Was it extraneous? Was it a mistake that it was here in the first place.

We need more context in PRs for changes like this.

Reply:
I do realize it moved into `create_group_activity`, but it took me a minute to see that, it would probably be helpful to have some annotation style notes for what was refactored and why